ledger-core/v2/network/consensus/gcpv2/phasebundle/ph3ctl/phase3.go

```diff
@@ -145,12 +148,16 @@ func (c *Phase3Controller) workerPhase3(ctx context.Context) {
 		defer runtime.UnlockOSThread()
 	}
 
+	phase3StartedAt := time.Now()
+
 	localInspector := c.workerPrePhase3(ctx)
 	if localInspector == nil {
 
 		// context was stopped in a hard way, we are dead in terms of consensus
 		// TODO should wait for further packets to decide if we need to turn ourselves into suspended state
 		// c.R.StopRoundByTimeout()
+		// TODO: low-latency metrics - https://insolar.atlassian.net/browse/PLAT-217
+		go stats.Record(ctx, metrics.Phase3Time.M(float64(time.Since(phase3StartedAt).Nanoseconds())*metrics.StatUnit))
```

Comment:
you can get rid of code duplication using `defer func() { go stats.Record(...) }`

Reply:
done